src/org/parosproxy/paros/view/View.java

```diff
@@ -220,6 +229,17 @@ public void init() {
 
         // adds the Request/Response representation buttons in ZAP toolbar
         getMessagePanelsPositionController().restoreState();
+        
+        String statusString = "";
+        for(Status status : AddOn.Status.values()) {     	
+        	//Try/catch in case AddOn.Status gets out of sync with cfu.status i18n entries
+        	try {
+        		statusString = Constant.messages.getString("cfu.status." + status.toString());
+        	} catch (MissingResourceException mre) {
+        		statusString = status.toString();

```

Comment:
I suggested error level because of the extra visibility when in dev mode, it's shown in the output tab and the bug count incremented.
We could also use error in dev mode and warn otherwise.